README.md

```diff
@@ -54,15 +54,17 @@ repositories {
  * Artifact:
 ```groovy
 dependencies {
-    compileOnly 'com.destroystokyo.paper:paper-api:1.16.5-R0.1-SNAPSHOT'
+    compileOnly 'io.papermc.paper:paper-api:1.17-R0.1-SNAPSHOT'
 }
 ```
 
 How To (Compiling Jar From Source)
 ------
-To compile Paper, you need JDK 8, maven, and an internet connection.
+To compile Paper, you need JDK 16 and an internet connection.
 
-Clone this repo, run `./paper jar` from *bash*, get files.
+Clone this repo, run `./gradlew applyPatches && ./gradlew reobfJar` from bash, get files from `Paper-Server/build/libs`.
```

Comment:
not sure I would recommend running this as one command, as if the first fails stuff might break, cause I think gradle doesnt always set exit codes